src/main.jsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react'
 import ReactDOM from 'react-dom'
-import App from './app'
+import App from './app.jsx'
```

Comment:
Opened a discussion on the topic -> https://github.com/opentable/eslint-config-opentable/issues/6